chromadb/api/types.py

```diff
@@ -477,7 +479,12 @@ def validate_embeddings(embeddings: Embeddings) -> Embeddings:
             f"Expected each embedding in the embeddings to be a list, got {embeddings}"
         )
     for embedding in embeddings:
-        if not all([isinstance(value, (int, float)) for value in embedding]):
+        if not all(
```

Comment:
Can you please normalize to our linter?

Reply:
This is the formatting that `pre-commit run` (black) gives me.